backend/utils/parseText.js

```diff
@@ -23,19 +34,21 @@ exports.getQueryDict = (text) => {
   nounTags = ["NN", "NNS", "NNP", "NNPS"];
 
   const taggedTokens = posTagger.tag(tokens);
-  let quantity, item;
+  let quantity = 0;
+  let item;
 
   for (let taggedToken of taggedTokens.taggedWords) {
     if (taggedToken.tag === "CD") {
-      quantity = parseInt(taggedToken.token);
-      if (isNaN(quantity)) {
-        quantity = wordsToNumbers.wordsToNumbers(taggedToken.token);
+      tempQuantity = parseInt(taggedToken.token);
+      if (isNaN(tempQuantity)) {
+          quantity += wordsToNumbers.wordsToNumbers(taggedToken.token)
+      } else {
+          quantity += tempQuantity
       }
     } else if (nounTags.includes(taggedToken.tag)) {
       item = nounInflector.singularize(taggedToken.token);
     }
   }
-  const dict = { item, quantity };
-  console.log({ text, dict });
```

Comment:
one small stuff, can you readd this log, would be helpful for debugging purposes in prod